extensions/dicom-sr/src/utils/addMeasurement.js

```diff
@@ -62,6 +62,12 @@ export default function addMeasurement(
   measurement.loaded = true;
   measurement.imageId = imageId;
   measurement.displaySetInstanceUID = displaySetInstanceUID;
+
+  // Remove the unneeded coord now its processed, but keep the SOPInstanceUID.
+  // NOTE: We assume that each SCOORD in the MeasurementGroup maps onto one frame,
```

Comment:
I think it has to, if we have the matching referenceid?

Reply:
Each SCOORD within a single TID 300 MeasurementGroup can map to a different image. There is no restriction as far as I can tell, but I've never seen it in practice yet.